client/src/components/plugin/index.js

```diff
@@ -1,7 +1,41 @@
-import youtube from './plugins';
+import RemarkIFrame from 'remark-iframes-extended';
 
-const plugins = [
-  youtube,
-];
+const defaultPluginProperties = {
+  default: {
+    tag: 'iframe',
+    width: 560,
+    height: 315,
+    disabled: false,
+    replace: [
+      ['http://', 'https://'],
+    ],
+    removeAfter: '&',
+  },
+};
 
-export default plugins;
+const youtubePluginProperties = {
```

Comment:
@Navneet-Sahota have you checked for responsiveness?

Reply:
Yes